CODEOWNERS

```diff
@@ -2,4 +2,16 @@
 # This file controls who is tagged for review for any given pull request.
 
 # For anything not explicitly taken by someone else:
-* @open-telemetry/dotnet-approvers
+* @open-telemetry/dotnet-contrib-approvers
+
+src/OpenTelemetry.Contrib.Exporter.Stackdriver/                             @open-telemetry/dotnet-contrib-approvers
```

Comment:
is it necessary to duplicate `dotnet-contrib-approvers`? Just curious

Reply:
Actually I'm not 100% sure. I guess this will notify any person and also the `dotnet-contrib-approvers` team when the PR is created for relevant path files. In the absence of `dotnet-contrib-approvers`, the PR will only notify the persons and only default to `dotnet-contrib-approvers` if certain paths have unassigned people.

I referred to how the opentelemetry-collector-contrib manages its [codeowners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS). If it works without the duplicate team mentions then we can remove it.